src/imitation/algorithms/tabular_irl.py

```diff
@@ -7,18 +7,46 @@
 
 .. _PhD thesis:
     http://www.cs.cmu.edu/~bziebart/publications/thesis-bziebart.pdf
+
+.. note::
+    Our current implementation of MCE IRL uses Jax, which is not bundled with the
+    default ``pip`` installation of ``imitation`` because Jax is incompatible with
+    Windows. You can install the Jax dependencies used by MCE IRL via
+    ``pip install imitation[jax]``.
+
+    We are considering porting MCE IRL from Jax to PyTorch in a future release.
 """
 
 import abc
 import logging
+import warnings
 
-import jax
-import jax.experimental.stax as jstax
-import jax.numpy as jnp
-import jax.random as jrandom
 import numpy as np
 import scipy
 
+try:
+    # pytype: disable=import-error
+    import jax
+    import jax.experimental.stax as jstax
+    import jax.numpy as jnp
+    import jax.random as jrandom
+
+    # pytype: enable=import-error
+
+except ImportError:
+    msg = (
+        f"Failed to import module {__name__} because Jax dependency is not installed. "
+        "See module docstring for more information on installation and OS "
+        "compatibility."
+    )
+    # ImportWarning is more appropriate than UserWarning here, but ImportWarning
+    # has been ignored by default since Python 3.7:
+    # https://docs.python.org/3/library/devmode.html#effects-of-the-python-development-mode
+    warnings.warn(msg)
```

Comment:
Why do we want this to be a warning? Is there a use case where we need to import `tabular_irl` but not use any of the functions? I'm inclined to just re-raise ImportError, but including the more informative error `msg` above.

Reply:
That sounds good to me. There's no reason to import this submodule passively (nothing other than `tests/` imports this module)